bsip-0083.md

```diff
@@ -0,0 +1,88 @@
+
+# Motivation
+Block production is an essential component of the BitShares platform itself, and as such should be expected to take an agnostic and neutral view towards the products deployed within the platform. Therefore, this BSIP will define a process for BitAsset management to ensure protocol security remains the singular focus of the block producer role and give BitAssets the best chance for long-term stability by nominating a new custodian to quickly react to market conditions. In order for the distributed consensus mechanism of DPOS to achieve adequate decentralization, it is imperative that block producers maintain strict independence from the politics of individual products deployed on the platform. 
+
+In short, this BSIP will put the BitAssets under the control of a new user account controlled by the stakeholders through a new type of voting. With this change, the newly appointed BitAssets manager will be able to independently maintain the parameters of their derivative asset without interference from the platform’s block producers and committee members. At the same time, the platform itself and its committee will not be unduly influenced by the demands of product management. An advantage gained by this is that the BitAsset manager will be able to react appropriately to rapidly changing market conditions, as may be required to properly maintain a successful derivative asset.
+
+# Rationale
+Because the BitAssets are currently controlled by platform-wide governance mechanisms, and because these are not well suited to the management of smart coins, which may at times need to respond quickly to market conditions, the authors propose to reorganize these assets to utilize a dedicated stakeholder-voted multisignature authority and thereby enable these assets to update in synchronization with market ebbs and flows.
+
+The block producers will no longer provide feeds for BitAssets. Instead another set of accounts will be authorized to provide feeds for BitAssets.  These accounts will be elected through stake-weighted voting with a new voting token called BITASSET.MANAGEMENT.  The initial distribution of this new voting token will be made equal to the current BTS distribution.  Subsequently, tokens of this new asset can be freely transferred or traded.
+
+Stake-weighted voting will also be used to elect a BitAsset Manager account (bitasset-manager) who will have the authority to change the settings of any BitAsset that are currently changeable by the Committee through proposals. The Committee will retain full control over the manager account, but is expected to leave day-to-day operations to the new elected entity.
+
+# Specification
+The following technical changes are required:
+
+**Assignment of Accumulated Fees**
+
+At the hardfork date, the accumulated fees of the existing BitAssets will be transferred to the committee-account.
+
+**Creation of BITASSET.MANAGEMENT asset**
```

Comment:
As mentioned in https://github.com/bitshares/bsips/pull/242#discussion_r336451636,
> A voting token would not have value if it tracks the voting weight of another token well (which is hard), in that case it is a simple utility token carries one role for another token. When we have this BSIP (token-based voting), I'd expect that people would directly vote with the tokens they already have but not create new tokens only for voting, because it would be economically much cheaper.
>
>This is also the reason that I don't agree with the MANAGEMENT token idea proposed in BSIP83, because the new token can't track the distribution of BTS well in the long run. That idea means to privatize committee-owned smartcoins, which is something similar to STEALTH, and would divide the community.

Reply:
Mentioned by @shulthz in https://github.com/bitshares/bsips/pull/238#discussion_r335265977:
>I don't think the “Creation of BITASSET.MANAGER asset” is necessary.
>
>This will also push the bitasset to be a third party, it's no different with before words.
>
>What's the right method is which the stakeholder of BTS vote the price feeder and committee of bitasset.

Reply:
These your are opinions. The authors have a different opinion. Approve the proposal and let the community decide.